code/modules/mining/equipment/explorer_gear.dm

```diff
@@ -95,3 +95,13 @@
 	var/mutable_appearance/glass_overlay = mutable_appearance(icon, "hostile_env_glass")
 	glass_overlay.appearance_flags = RESET_COLOR
 	add_overlay(glass_overlay)
+
+/obj/item/clothing/suit/space/hostile_environment/equipped(mob/living/carbon/human/user, slot)
+  if(slot == slot_wear_suit)
+    flags |= NODROP
```

Comment:
Why make this nodrop when worn? Doesn't that mean that once eqipped, the suit acts are unremovable armor, and potentially prevents anyone removing say your headset underneath it?

Reply:
wouldn't it also prevent surgery?

Reply:
It will not prevent surgery. I made the suit nodrop when worn because it is meant to be a cursed item. I have considered how this would make the wearer KOS in terms of space law, because no stripping and such. However I had not considered how this would affect antags Vs the wearer. Not being able to remove someone's headset would make the suit very powerful against vampires and anyone else who tries to abduct the wearer. There is two things that I could do. I could remove the nodrop when worn on the helmet, so antags can actually take the wearer's headset while still retaining the suits curse. Or I could leave it how it is and make the entire suit nodrop, this might have balancing conserns. However I think these conserns aren't too bad as a pAI can also make the user unabductable.

Reply:
Don't use NODROP for it. There are just too many situations where having NODROP clothing is an advantage, particularly if the clothing also counts as armor/suit.
Instead, make it removable, but have it apply some sort of horrible curse to the user IF it is removed. Like killing off 1-2 of your internal organs.

Reply:
how about if it was unremovable but if you have holy water in your system you can remove it?

Reply:
I'd really rather you just give it some other downside that isn't "hard to remove".

Reply:
I am thinking of scrapping this part, then in a future pr I might flesh out the heck suit. I am a very inexperienced coder and I'd rather not take on more than I can chew